models/articles.model.js

```diff
@@ -1,5 +1,41 @@
 const db = require("../db/connection.js");
 
+function selectArticles(topic) {
+
+    const validTopics = ["mitch", "cats"];
```

Comment:
Hey Tim if we leave this like this, each time a user adds a new topic (I know we dont have that yet) we would have to change the code for valid topics, this isbn going to be viable going forwards, we looked at how we could solve this in yesterdays lecture, so have a look at the error handling notes and then have a go at refactoring